docs/appguide.rst

```diff
@@ -49,6 +49,8 @@ Some of the key points of interest in this example application are:
    a common wasp-os idiom that is normally used to pattern to distinguish a full
    refresh of the screen from an fast update (a redraw).
 
+The full list of application entry points can be found at the bottom of this page.
+
```

Comment:
Cross-linking here is wrong.

This material is strictly introductory and the user is not yet ready to make a decision to follow this cross-link or not.

Instead, if the API entry points are the next thing we think the reader should read then that section should be physically moved here. If the Application life-cycle section is better next material to read then we don't need a cross-link.

Cross links are important to navigating a document but they should never be a "workaround" for a document that is simply in the wrong order!

Reply:
I moved the Application entry point list higher up

Reply:
Let's remove this paragraph. It's no longer correct (since it is not at the bottom of the page) and with the new ordering it is not useful to cross-link here.